src/main/java/Application.java

```diff
@@ -0,0 +1,39 @@
+import controller.CarNameException;
+import controller.RacingCarController;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Stream;
+import model.RacingCar;
+import view.RacingCarView;
+import view.TryCountException;
+
+public class Application {
+
+    public static void main(String[] args) throws CarNameException, TryCountException {
+        RacingCarView racingCarView = new RacingCarView();
+        RacingCarController racingCarController = new RacingCarController();
+
+        racingCarView.inputCarName();
+
+        String[] carNames = racingCarView.getCarName();
+        racingCarController.checkCarNamesValid(carNames);
+        List<String> carNamesList = Arrays.asList(carNames);
+        Stream<String> carNamesStream = carNamesList.stream();
+
+        racingCarView.inputTryCount();
+        int tryCount = racingCarView.getTryCount();
+
+        ArrayList<RacingCar> racingCars = new ArrayList<>();
+        carNamesStream.forEach(carName -> racingCars.add(new RacingCar(carName)));
```

Comment:
```suggestion
        racingCarView.inputTryCount();
        int tryCount = racingCarView.getTryCount();

        List<String> carNamesList = Arrays.asList(carNames);
        Stream<String> carNamesStream = carNamesList.stream();
        ArrayList<RacingCar> racingCars = new ArrayList<>();
        carNamesStream.forEach(carName -> racingCars.add(new RacingCar(carName)));
```
관련된 코드들을 모아두는 것이 가독성을 더 나아지게 만듭니다.

`map`과 `collect` 등을 활용한 메소드 체이닝을 사용하면 좀 더 깔끔한 코드를 만들 수 있습니다.

```java
        List<RacingCar> racingCars = Arrays.stream(carNames)
                .map(RacingCar::new)
                .collect(Collectors.toList());
```